app/assets/javascripts/helpers/post_photo_uploader.es6

```diff
@@ -71,6 +76,22 @@ Diaspora.PostPhotoUploader = class {
         onError: (id, name, errorReason) => this.showMessage("error", errorReason)
       }
     });
+
+    if (this.dropZoneElementIds !== undefined && this.dropZoneElementIds instanceof Array) {
+      var dropZoneElements = this.dropZoneElementIds.map(id => document.getElementById(id))
+        .filter(el => el !== undefined && el !== null);
+      if (dropZoneElements.length > 0) {
+        var dndFineUploader = this.fineUploader;
+        this.dragAndDropModule = new qq.DragAndDrop({
+          dropZoneElements: dropZoneElements,
+          callbacks: {
+            processingDroppedFilesComplete: function(files) {
```

Comment:
Since we use the latest js syntax in this file, you can use the arrow syntax and make it a one-liner:
`processingDroppedFilesComplete: files => this.fineUploader.addFiles(files)`

Reply:
Sounds good.